Makefile

```diff
@@ -56,7 +56,7 @@ update-vendor:
 .PHONY: tools
 tools:
 	docker pull golang:1.9-alpine
-	docker pull chai2010/grpc-tools
+	docker pull openpitrix/openpitrix:api-builder
```

Comment:
We have such tag for the docker?

Reply:
https://hub.docker.com/r/openpitrix/openpitrix/tags/
It is created by @chai2010, has already associated with ower github project. I just add api-builder to the build setting menu.